style/rst.rst

```diff
@@ -0,0 +1,464 @@
+==============
+RST Guidelines
+==============
+
+At Crate.io, our documentation is written in RST. In order to create a uniform
+standard for writing our RST, we have developed some guidelines. Following
+these guidelines is required for contributions to our documentation. For ease
+of use, we have gathered them in this wiki page below.
+
+**Note**:
+
+    Due to the limitations of Github's RST support, some of the guidelines as
+    written here do not themselves correspond to best practice. For example, a
+    note like this should use an RST admonition and consistent pre-spacing.
+    This applies also to support for distinguishing open and closed lists. It
+    is better to follow the spirit of this guide than the RST of this specific
+    document!
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+
+.. _rst-general:
+
+General
+=======
+
+With the exception of pre-formatted text (e.g., literals and code blocks), as
+well as link URLs, lines should not exceed 79 characters. They should be
+wrapped as closely to 79 characters per line as possible, without exceeding
+that limit.
+
+**Tip**:
```

Comment:
We can also now use Sphinx theme things for tips, warnings, etc again (I just used bold for the wiki)